tests/test_decorators.py

```diff
@@ -30,6 +31,8 @@
 
 api = hug.API(__name__)
 module = sys.modules[__name__]
+test_directory = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I would suggest, just changing these to TEST_DIRECTORY and BASE_DIRECTORY, if they are immutable constants, otherwise looks good!